plugins/org.obeonetwork.m2doc.doc/doc/user/M2Doc User Guide.html

```diff
@@ -173,6 +173,18 @@ <h4 id="Siriustableproviders">Sirius table providers</h4>
 			<b>Warning</b>: do not forget the single quotes ' around the string in an AQL Expression: 
 			<code>title:"'Table name'"</code>. The description ID does not need such quotes because it is of type String.
 		</p>
+		<h4 id="UpdatestrategyforSiriusRepresentation">Update strategy for Sirius Representation</h4>
+		<p>In the cases of generating a diagram with {m:diagram} or a table with {m:wtable}, the M2Doc generation can take place on an unsynchronized representation (diagram or table). Consequently, a parameter has been introduced in the genConf model to allow users to specify whether representations must be refreshed before the M2Doc generation.</p>
+		<p>
+			<img border="0" src="images/image019.png"/>
+		</p>
+		<p>The boolean parameter &#171;refreshRepresentation&#187; can be used to define following update strategy : 
```

Comment:
I change all reference of 'refreshFlag' in code by 'refreshRepresentation'  or refreshRepresentationFlag